titan-server/src/models.rs

```diff
@@ -432,4 +433,43 @@ pub struct UserEventExcuseWithAssoc {
     pub ack_comments: Option<String>,
     pub date_created: chrono::NaiveDateTime,
     pub date_modified: chrono::NaiveDateTime,
-}
\ No newline at end of file
+}
+
+#[derive(Identifiable, Serialize, Deserialize, Queryable)]
+pub struct Report {
+    pub id: i32,
+    pub role_id: i32,
+    pub term_start_date: chrono::NaiveDateTime,
+    pub submission_date: Option<chrono::NaiveDateTime>,
+    pub comments: Option<String>,
+    pub ack_user_id: Option<i32>,
+    pub ack_date: Option<chrono::NaiveDateTime>,
+    pub date_created: chrono::NaiveDateTime,
+    pub date_modified: chrono::NaiveDateTime,
+}
+
+#[derive(Insertable)]
+#[table_name = "reports"]
+pub struct NewReport {
```

Comment:
This is a premature addition given that the issue regards just listing reports. It makes sense to add because we'll need it for the next issue, but it's worth pointing out that this isn't really related to the issue at hand.